crypto/bn/bn_test.cc

```diff
@@ -171,6 +172,66 @@ static ScopedBIGNUM GetBIGNUM(FileTest *t, const char *attribute) {
   return ret;
 }
 
+// TODO Remove as dependency, per comment at top of file
```

Comment:
Please put this in bn_test_lib.c. (To determine which of bn_test_lib.c or bn_test_convert.c to move the code too, I diff a diff of the licenses at the top of random.c and those two functions. bn_test_lib.c has the same license so that's where it goes.)